app/utils/auth/providers.py

```diff
@@ -400,3 +400,31 @@ def get_userinfo(cls, user: models_core.CoreUser):
             "email": user.email,
             "is_admin": is_user_member_of_any_group(user, [GroupType.admin]),
         }
+
+
+class PlankaAuthClient(BaseAuthClient):
+    """
+    An auth client for Planka, a Trello alternative for kanban boards
+
+    Docs for OIDC integration:
+    https://docs.planka.cloud/docs/Configuration/OIDC/
+    """
+
+    allow_pkce_with_client_secret: bool = True  # required to exchange OIDC code
```

Comment:
To be honest I'm not sure if I understood correctly the comment. What did you mean?
```suggestion
    allow_pkce_with_client_secret: bool = True
```

Reply:
In practice, if this boolean is left to false, Planka throws an error right after the redirection from CalypSSO.
It says that (IIRC) the "exchange of the OIDC codes using PKCE" failed.
This message appears both in Planka's logs and right after the callback in the response to a failed request.

This led me to look at this boolean about PKCE.
When I set it to true, the authentication process worked as expected.